src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_index_pattern/step_index_pattern.html

```diff
@@ -1,169 +1,170 @@
-            ng-click="stepIndexPattern.goToNextStep()"
-            ng-disabled="!stepIndexPattern.canGoToNextStep()"
-          >
-            <span class="kuiButton__inner">
-              <span>
-                Next step
-              </span>
-              <span
-                aria-hidden="true"
-                class="kuiButton__icon kuiIcon fa-chevron-right"
-              ></span>
-            </span>
-          </button>
-        </div>
-      </form>
-
-      <!-- List of matching indices -->
-      <div
-        class="kuiVerticalRhythm"
-        ng-switch="stepIndexPattern.matchingIndicesListType"
+  <form
+    name="stepIndexPattern.indexPatternNameForm"
+    role="form"
+    ng-submit="stepIndexPattern.goToNextStep()"
+  >
+    <div class="euiFormRow">
+      <label
+        for="indexPatternNameField"
+        class="euiFormLabel"
       >
-        <div ng-switch-when="invalidIndexPattern">
-          <matching-indices-list
-            is-loading="stepIndexPattern.isFetchingMatchingIndices"
-            indices="stepIndexPattern.allIndices"
-            pattern="stepIndexPattern.indexPatternName"
-          >
-           <span class="kuiStatusText kuiStatusText--error">
-              <span
-                aria-hidden="true"
-                class="kuiStatusText__icon kuiIcon fa-hand-stop-o"
-              ></span>
-
-              <span>
-                You've entered an invalid index pattern. Please adjust it to match any of your <strong>{{stepIndexPattern.allIndices.length}} indices</strong>, below.
-              </span>
-            </span>
-          </matching-indices-list>
-        </div><div ng-switch-when="noInput">
-          <matching-indices-list
-            is-loading="stepIndexPattern.isFetchingMatchingIndices"
-            indices="stepIndexPattern.allIndices"
-            pattern="stepIndexPattern.indexPatternName"
-          >
-            <span class="kuiStatusText kuiStatusText--info">
-              <span ng-if="!stepIndexPattern.allIndices.length">
-                You only have a single index. You can create an index pattern to match it.
-              </span>
+        Index pattern
+      </label>
+      <div class="euiFormControlLayout">
+        <!-- Index pattern input -->
+        <input
+          id="indexPatternNameField"
+          class="euiFieldText euiFieldText--fullWidth createIndexPatternInputField"
+          data-test-subj="createIndexPatternNameInput"
+          ng-model="stepIndexPattern.indexPatternName"
+          placeholder="index-name-*"
+          validate-index-pattern
+          validate-index-pattern-allow-wildcard
+          append-wildcard
+          name="name"
+          required
+          type="text"
+          aria-describedby="indexPatternNameFieldHelp1 indexPatternNameFieldHelp2"
+        >
+      </div>
+    </div>
+    <div class="euiSpacer euiSpacer--xs"></div>
+    <div class="euiFlexGroup">
+      <div class="euiFlexItem">
+        <p
```

Comment:
Let's move these two paragraphs into the help text of the above `euiFormRow`.

Reply:
Once we do that, we can wrap the form row and the action button in a `euiFlexGroup` to get the alignment right. Pseudo-code showing what I mean:

```html
    <div class="euiFlexGroup euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--alignItemsEnd euiFlexGroup--responsive">
      <div class="euiFlexItem euiFlexItem--flexGrowZero">
        <div class="euiFormRow">
          <!-- Form row stuff -->
        </div>
      </div>

      <div class="euiFlexItem euiFlexItem--flexGrowZero">
        <!-- Action -->
        <button>
          Next step
        </button>
      </div>
    </div>
```

It will look roughly like this:

![image](https://user-images.githubusercontent.com/1238659/34230038-7b630a76-e58b-11e7-9f70-2598770cfd8b.png)